lib/coxir/struct/user.ex

```diff
@@ -102,6 +102,23 @@ defmodule Coxir.Struct.User do
     |> Channel.pretty
   end
 
+  @doc """
+  Sends a DM to a given user.
+
+  Returns a message object upon success
+  or a map containing error information.
+  """
+  @spec send_message(user, String.t) :: map
+
+  def send_message(%{id: id}, content),
+    do: send_message(id, content)
+
+  def send_message(recipient, content) do
+    create_dm(recipient)
```

Comment:
Can we turn this into a pipe?

Reply:
Also, at this point we are assuming `create_dm/1` is never going to fail.

Are we safe doing so?